databroker/src/grpc/kuksa_val_v2/conversions.rs

```diff
@@ -45,7 +45,10 @@ impl From<&proto::Datapoint> for broker::Datapoint {
 impl From<broker::Datapoint> for Option<proto::Datapoint> {
     fn from(from: broker::Datapoint) -> Self {
         match from.value {
-            broker::DataValue::NotAvailable => None,
+            broker::DataValue::NotAvailable => Some(proto::Datapoint {
```

Comment:
This one is worth discussing. Previously (for V2) we  just ignored "NotAvailable", i.e. it was never returned in subscribe. But at lest for GetValue we want to return it, so I move the filtering higher up.

But how do we want it to work for Subscribe? If we intend to support actively removing a DataPoint by providing a DataPoint with value Null, then maybe we want the subscribe to also have support for returning Null.

And if so, would it not make sense to also return Null in the first request when subscription is started?

(Currently PR does not change behavior for Subscribe)

Reply:
Discussed among committers - it seems we want None to be returned also for subscribe. I will check how it affects test case and update the PR

Reply:
It returns `None` there, but as it returns `None` there it means that the value will be discarded (and not sent to client) at https://github.com/eclipse-kuksa/kuksa-databroker/blob/feature/databroker-api-v2/databroker/src/grpc/kuksa_val_v2/val.rs#L638

Thats why we need to create a Datapoint with value `None` here.